tasks/nsp.js

```diff
@@ -21,7 +21,12 @@ module.exports = function (grunt) {
     }
 
     if (config.output) {
-      formatter = Nsp.formatters[config.output];
+      var formatters = Object.keys(Nsp.formatters);
+      if (formatters.indexOf(config.output) !== -1) {
+        formatter = Nsp.formatters[config.output];
+      } else {
+        grunt.log.write('Invalid formatter specified in config. Must be one of ' + Object.keys(Nsp.formatters).join(',') + '\n');
```

Comment:
You can replace `Object.keys(Nsp.formatters)` here with `formatters` (from line 24).